leapp/messaging/answerstore.py

```diff
@@ -46,10 +46,10 @@ def _load_ini(cls, inifile):
         Loads an ini config file from the given location.
 
         :param inifile: Path to the answer file to load.
-        :return: configparser.SafeConfigParser object
+        :return: configparser.ConfigParser object
         :raises CommandError if any of the values are not in key=value format
         """
-        conf = configparser.SafeConfigParser(allow_no_value=False)
+        conf = configparser.ConfigParser(allow_no_value=False)
```

Comment:
This should be alright, SafeConfigParser has been an alias for ConfigParser since Python 3.2 (see https://docs.python.org/3/whatsnew/3.2.html#configparser for more)

Reply:
I thought we could do it as I do not believe we will rebase leapp in RHEL 7 anymore. however, if we want to run tests for py2 another year, I guess we should keep py2 compatibility in the framework too, until we officially drop it. Most likely this change will not affect tests, but something is telling me it would be better to put there `if-else` statement to check what python is running.

@oamg/developers  any thoughts?

Reply:
I agree with keeping it py2 compatible for now but lets do the conditional stuff at the top, when we import, rather than sprinkling changes inside of the main body of code.  That way it's easier to find the places that can have the compatibility removed later.  (Another alternative, if there are a lot of places, is to put all compat code into a separate file and import from there.)

Reply:
@abadger Well I could put it at the top, but the imports are not very nice. I prefer `if sys.version < (3, 2) ... else ...` inside the function. I think we can grep these checks easily.
Also, we have leapp/compat.py, we could put it there, but I am not sure if that's worth it.

Reply:
The first comment was about whether ConfigParser is the right replacement, we had offline discussion about that with @Rezney

Reply:
Why aren't the imports nice?

At worst, you end up doing the `if sys.version` check there. (But `try: except:` is generally better as it's a feature check.)

Reply:
Well the nicest I could come up with is:
```
try:
    from six.moves.configarser import SafeConfigParser as Configparser
except ImportError:
    from six.moves.configarser import ConfigParser
```
but I don't

Reply:
@abadger I changed it, wdyt?